php-checks/src/main/java/org/sonar/php/checks/HttpOnlyCheck.java

```diff
@@ -79,4 +91,25 @@ private static boolean isSetCookie(FunctionCallTree tree) {
     String functionName = getLowerCaseFunctionName(tree);
     return functionName != null && SET_COOKIE_FUNCTIONS.contains(functionName);
   }
+
+  private boolean isSymfonyCookieCreation(FunctionCallTree tree) {
+    ExpressionTree callee = tree.callee();
+
+    if (callee.is(CLASS_MEMBER_ACCESS)) {
+      MemberAccessTree memberAccessTree = (MemberAccessTree) callee;
+      ExpressionTree receiver = memberAccessTree.object();
+      String method = nameOf(memberAccessTree.member());
+      return "create".equals(method) && receiver.is(Kind.NAMESPACE_NAME) && SYMFONY_COOKIE.equals(getFullyQualifiedName((NamespaceNameTree) receiver));
+    }
+    if (callee instanceof ClassNamespaceNameTreeImpl) {
+      return ((ClassNamespaceNameTreeImpl) callee).symbol().qualifiedName().equals(SYMFONY_COOKIE);
+    }
+    return false;
+  }
+
+  private void createIssueIfHttpOnlyIsFalse(ExpressionTree argument, FunctionCallTree tree) {
+    if (argument.is(Kind.BOOLEAN_LITERAL) && isFalseValue(argument)) {
```

Comment:
can we rely on the isFalseValue function only ? should we catch the cases where developer use 0 or "false" instead of false ?
```suggestion
    if (isFalseValue(argument)) {
```

Reply:
it's probably a good idea, I will change this